src/components/DocumentRenderer.tsx

```diff
@@ -0,0 +1,68 @@
+import { FrameActions, FrameConnector, HostActionsHandler } from "@govtechsg/decentralized-renderer-react-components";
+import { getData, WrappedDocument } from "@govtechsg/open-attestation";
+import React, { useState, useCallback, useEffect } from "react";
+
+interface DocumentRendererProps {
+  rawDocument: WrappedDocument;
+}
+
+export const DocumentRenderer: React.FunctionComponent<DocumentRendererProps> = ({
+  rawDocument,
+}: DocumentRendererProps) => {
+  const document = getData(rawDocument);
+  const [templates, setTemplates] = useState([]);
```

Comment:
templates is not correctly types, because it's not used at the moment => get rid of everything related to templates